infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -414,30 +416,40 @@ private void registerToLogReplicationLock() {
      * as source (sender/producer) or sink (receiver).
      */
     private void onLeadershipAcquire() {
-        switch (topologyDescriptor.getLocalClusterDescriptor().getRole()) {
-            case SOURCE:
-                log.info("Start as Source (sender/replicator)");
-                sessionManager.startReplication();
-                lockAcquireSample = recordLockAcquire(topologyDescriptor.getLocalClusterDescriptor().getRole());
-                processCountOnLockAcquire();
-                break;
-            case SINK:
-                log.info("Start as Sink (receiver)");
-
-                // Sink Site : the LogReplicationServer (server handler) will
-                // reset the LogReplicationSinkManager on acquiring leadership
-                interClusterServerNode.setLeadership(true);
-                lockAcquireSample = recordLockAcquire(topologyDescriptor.getLocalClusterDescriptor().getRole());
-
-                processCountOnLockAcquire();
-                break;
-            default:
-                log.error("Log Replication not started on this cluster. Leader node {} id {} belongs to cluster with {} role.",
-                        localEndpoint, localNodeId, topologyDescriptor.getLocalClusterDescriptor().getRole());
-                break;
+        // TODO [V2]: a cluster can Source and/or Sink for different replication models. The support for this is coming
```

Comment:
Nit: comment: can -> can be
Are these comments under 120 char limit?

Reply:
Yes they were within the limit. now 2 characters are beyond 120 chars, moving "coming" to the next line.